tests/baselines/reference/commentsExternalModules.js

```diff
@@ -129,6 +129,8 @@ define(["require", "exports"], function (require, exports) {
 define(["require", "exports", "commentsExternalModules_0"], function (require, exports, extMod) {
     "use strict";
     Object.defineProperty(exports, "__esModule", { value: true });
+    /**This is on import declaration*/

```

Comment:
This doesn't look right.. This jsdoc comment is associated with import declaration and without it (and missing trailing comments followed by another declaration) could make these associated with that declaration...